app/components/lettings_log_summary_component.html.erb

```diff
@@ -1,27 +1,21 @@
 <article class="app-log-summary">
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-column-two-thirds">
       <header class="app-log-summary__header">
         <h2 class="app-log-summary__title">
           <%= govuk_link_to lettings_log_path(log) do %>
             Log <%= log.id %>
           <% end %>
         </h2>
         <% if log.tenancycode? or log.propcode? %>
-          <dl class="app-metadata app-metadata--inline">
+          <div class="app-metadata">
             <% if log.tenancycode? %>
-              <div class="app-metadata__item">
-                <dt class="app-metadata__term">Tenant</dt>
-                <dd class="app-metadata__definition"><%= log.tenancycode %></dd>
-              </div>
```

Comment:
Do you think description list tag might be more informative given the tenant/property codes here are metadata?

Reply:
I see what you mean, but using these tags led to some surprirising behaviour because the tag and descriptions were being formatted as separate blocks - e.g. 
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/2638ae64-fc8d-4d31-b493-b0039dcf3ab4)

I could spend some time fiddling with CSS to get around this, but the issue is that we don't really want them to be separate - we want the tag and description to be associated with each other in the same string, so that it wraps nicely (and e.g. the tenancy code can wrap under the word Tenant, rather than wrapping in its own container)